### PR TITLE
update get_all_workouts to pull in more than 20 workouts

### PR DESCRIPTION
**Pull request by dusty-turner**
Hi, I was experimenting with the Peloton API when I came across your repository.  I noticed that your get workouts function only returned 20 workouts and you had a comment about figuring out how to add more.  I worked through it and thought I could propose a simple change that would help improve your great package.  Thanks!

### CODE REVIEW

File: R/queries.R [hunk 1]
```diff
@@ -62,10 +62,9 @@ get_perfomance_graphs <- function(workout_ids, every_n = 5) {
 #' get_all_workouts()
 #' }
 #'
-get_all_workouts <- function(userid = Sys.getenv("PELOTON_USERID")) {
-  # TODO pagination  in API ?
-  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", call. = FALSE)
-  workouts <- peloton_api(path = glue::glue("api/user/{userid}/workouts"))
+get_all_workouts <- function(userid = Sys.getenv("PELOTON_USERID"), num_workouts = 20) {
+  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", ca, ll. = FALSE)
```
Comment (bweiher) [inline R/queries.R]:
```suggestion
  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", call. = FALSE)
```
looks like there is a typo here

**Comment by bweiher** [comment]
Thank you for the PR! LGTM!

Do you happen to know if the `&limit=N`  parameter has a capped value (e.g. 100), or will it always return N workouts (assuming you've done that many rides)?  I'm only around 23 rides but tested your changes and they worked well for me but curious what happens if someone has hundreds, etc.

File: R/queries.R (hunk 1 above)
Comment (dusty-turner) [reply R/queries.R]: You are right - must have messed that up after I checked it locally.  Thanks for that catch.

**Comment by dusty-turner** [comment]
So I just checked and it will error if you requested more workouts than you've completed.  Let me try to work on some error handling.

**Pull request merged**